storage/redis.go

```diff
@@ -59,52 +59,133 @@ func (r *redisStore) CheckRtt(ctx context.Context) int64 {
 
 var sidekiqHeartbeatJobsKey = ":work"
 
-func (r *redisStore) SendHeartbeat(ctx context.Context, heartbeat *Heartbeat) error {
+func (r *redisStore) SendHeartbeat(ctx context.Context, heartbeat *Heartbeat, heartbeatManagerTTL time.Duration) error {
```

Comment:
I would make this TTL part of the `Heatbeat` struct and extract the value just like the other values below.

Reply:
Refactored in fixup: 8ae85e1c6e9047312b29e23e4256a882d7272e08

Reply:
Thank you for updating this.